README.md

```diff
@@ -69,8 +69,8 @@ disk you just created. Allow any host to connect[^simple].
 
     modprobe nvmet nvmet-tcp
     cd /sys/kernel/config/nvmet/subsystems
-    mkdir $NQN
```

Comment:
In the Redhat POC I do all of this with a pre-configured tcp.json file.  FYI.

Reply:
This is the "very basic developer level" part of the SUSE PoC. In the "real" PoC, it's all done by scripts, too. But I had got the feedback from folks here at SUSE that they much disliked the scripted server setup. They wanted to see the gory details. Therefore I added this section.